src/main/java/org/apache/brooklyn/cloudfoundry/location/CloudFoundryClientRegistryImpl.java

```diff
@@ -28,17 +28,17 @@
 import org.apache.brooklyn.util.core.config.ConfigBag;
 import org.cloudfoundry.client.lib.CloudCredentials;
 import org.cloudfoundry.client.lib.CloudFoundryClient;
+import org.cloudfoundry.client.lib.CloudFoundryOperations;
 
+public class CloudFoundryClientRegistryImpl implements CloudFoundryClientRegistry {
 
-public class CloudFoundryPaasClientRegistryImpl implements CloudFoundryClientRegistry {
+    public static final CloudFoundryClientRegistryImpl INSTANCE = new CloudFoundryClientRegistryImpl();
 
-    public static final CloudFoundryPaasClientRegistryImpl INSTANCE = new CloudFoundryPaasClientRegistryImpl();
-
-    protected CloudFoundryPaasClientRegistryImpl() {
+    protected CloudFoundryClientRegistryImpl() {
     }
 
     @Override
-    public CloudFoundryPaasClient getCloudFoundryClient(ConfigBag conf, boolean allowReuse) {
+    public CloudFoundryOperations getCloudFoundryClient(ConfigBag conf, boolean allowReuse) {
```

Comment:
cool, it should be easy to switch to `CloudFoundryClient` when you upgrade the `cf-java-client`